lib/lightning_web/live/run_live/components.ex

```diff
@@ -155,7 +155,7 @@ defmodule LightningWeb.RunLive.Components do
     </.toggle_bar>
     <%= if @show_input_dataclip do %>
       <div id="input_section" style="display: none;" class="@container">
-        <.dataclip_view dataclip={@run.input_dataclip} />
+        <.dataclip_view dataclip={@run.input_dataclip} run={@run} />
       </div>
```

Comment:
@wanecode I think instead of passing the run into the dataclip view, we should have the markup and logic here instead. I'd like to avoid adding coupling between dataclips and runs here - ideally the `dataclip_view` component has no idea about a run.

We could take the existing code further as well and have the `dataclip_view` component expect a dataclip, but I think we should at least try and avoid increasing the number of dependencies.